mteb/abstasks/AbsTaskRetrieval.py

```diff
@@ -130,4 +139,6 @@ def encode_corpus(self, corpus: List[Dict[str, str]], batch_size: int, **kwargs)
                 (doc["title"] + self.sep + doc["text"]).strip() if "title" in doc else doc["text"].strip()
                 for doc in corpus
             ]
+        if prefix != '':
+            sentences = [prefix + sentence for sentence in sentences]
```

Comment:
I think we can remove this. This should be done in the encode method of the model. If the model has an `encode_corpus` function then it will automatically use that

Reply:
I agree with your suggestion to remove this section. Thanks for pointing this out!